src/goto-instrument/dump_c.cpp

```diff
@@ -593,7 +593,7 @@ void dump_ct::cleanup_decl(
   tmp.add(goto_programt::make_decl(decl.symbol()));
 
   if(value.is_not_nil())
-    tmp.add(goto_programt::make_assignment(code_assignt(decl.op0(), value)));
+    tmp.add(goto_programt::make_assignment(decl.op0(), value));
```

Comment:
While updating this line, could we also use `decl.symbol()` instead of `decl.op0()`

Reply:
done